rollup.config.esm.js

```diff
@@ -0,0 +1,26 @@
+const commonjs = require('rollup-plugin-commonjs');
+const license = require('rollup-plugin-license');
+
+module.exports = {
+  input: 'src/marked.js',
+  output: {
+    file: 'lib/marked.esm.js',
+    format: 'esm'
+  },
+  plugins: [
+    license({
+      banner: `
+DO NOT EDIT THIS FILE
+The code in this file is generated from files in ./src/
+`
+    }),
+    license({
+      banner: `
+marked - a markdown parser
+Copyright (c) 2011-2018, Christopher Jeffrey. (MIT Licensed)
```

Comment:
```suggestion
Copyright (c) 2011-2019, Christopher Jeffrey. (MIT Licensed)
```

Reply:
I was thinking about just leaving `2011` so we don't need to update the year every year. Or we could change it to make the year update automatically.

Reply:
Good point, I updated my suggestion above